src/components/Launch/LaunchWorkflowForm/useLaunchWorkflowFormState.ts

```diff
@@ -221,10 +221,7 @@ export function useLaunchWorkflowFormState({
     >();
     const selectedWorkflowId = selectedWorkflow ? selectedWorkflow.data : null;
 
-    const inputValueCache = useMemo(
-        () => createInputValueCache(initialParameters.values),
-        [initialParameters.values]
-    );
+    const [inputValueCache] = useState(createInputValueCache());
```

Comment:
Were you running into issues with `useMemo`? I guess I'm struggling to see why we'd use `useState` here when the value doesn't seem to be mutated ever

Reply:
Nope, no issues. `useMemo` is overly complicated here now. We don't insert any values into the cache anymore at this level. So we just want to create it exactly once and reuse the value for the life of the component.